internal/docker/docker.go

```diff
@@ -453,19 +454,34 @@ func (handler *Handler) ContainerRemove(ctx context.Context, srcContainerID stri
 
 // ProjectDir returns the project directory as is configured for the given image.
 func (handler *Handler) ProjectDir(ctx context.Context, imageID string) (string, error) {
-	ii, _, err := handler.client.ImageInspectWithRaw(ctx, imageID)
+	// The image can tell us via a label where saucectl should mount the project files.
+	// We default to the working dir of the container as the default mounting target.
+	return handler.getObjectLabel(ctx, imageID, "com.saucelabs.project-dir")
+}
+
+// ChromeVersion returns the version of chrome installed in the given imageID
+func (handler *Handler) ChromeVersion(ctx context.Context, imageID string) (string, error) {
+	return handler.getObjectLabel(ctx, imageID, "selenium_chrome_version")
+}
+
+// FirefoxVersion returns the version of firefox installed in the given imageID
+func (handler *Handler) FirefoxVersion(ctx context.Context, imageID string) (string, error) {
+	return handler.getObjectLabel(ctx, imageID, "selenium_firefox_version")
+}
+
+func (handler *Handler) getObjectLabel(ctx context.Context, objectID string, label string) (string, error) {
```

Comment:
Imho this naming shows a clearer intent. After all, you are only inspecting images in there for their labels.
```suggestion
func (handler *Handler) getImageLabel(ctx context.Context, imageID string, label string) (string, error) {
```